src/drive/mobile/styles/feedback.styl

```diff
@@ -13,3 +16,9 @@
 
 .button-block
     text-align right
+
+.c-btn--paperplane
```

Comment:
I guess this should belong in Cozy-UI at some point.

Reply:
Weeeell I don't know, probably not. Have I had more time, I would just have used the `Icon` component which is a better solution for this IMO. But it involves changing the webpack configuration and I don't feel confident doing that without breaking anything just before the sprint end.

Reply:
Either way, it's ok to let this as is, don't worry. ;)